gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/configurations/DataFixConfiguration.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.swe.runner.app.configurations;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * @author Sebastien Murgey {@literal <sebastien.murgey at rte-france.com>}
+ */
+@Configuration
+@ConfigurationProperties(prefix = "swe-runner.data-fix")
+public class DataFixConfiguration {
```

Comment:
This Class could be replaced by a Record:
```java
@ConfigurationProperties(prefix = "swe-runner.data-fix")
public record DataFixConfiguration(boolean removeRemoteVoltageRegulationInFrance) {
}
```

This change requires to remove the `@Configuration` annotation on the class and to change annotation in SweApplication class as follows:
```java
@EnableConfigurationProperties({DataFixConfiguration.class, UrlConfiguration.class})
```
   
   
   
Same should be done with other configuration classes, but it is not the point of this PR...